apps/assisted_tagging/models.py

```diff
@@ -93,7 +93,8 @@ class PredictionStatus(models.IntegerChoices):
     class Type(models.IntegerChoices):
         AUTO = 0, 'Auto Extraction'  # NLP defiend extraction text
         MANUAL = 1, 'Manual Extraction'  # manual defined extraction text
-
+    page = models.IntegerField(default=0)
+    textOrder = models.IntegerField(default=0)
```

Comment:
Can you add some note on what this represents?
For python, always use snake case.
```suggestion
    page = models.IntegerField(default=0)
    text_order = models.IntegerField(default=0)
```